File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    var answer = 0;
    var a= (order+"").split("");
    for(let i=0;i<a.length;i++){
        if(a[i]=="3"||a[i]=="6"||a[i]=="9"){
            answer++
        }
    }
    return answer;
}

File: 프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        String[] orderString = (order+ "").split("");
        for(int i = 0; i< orderString.length; i++){
            if(orderString[i].equals("3") || orderString[i].equals("6") || orderString[i].equals("9")){
                answer++;
            }
        }
        return answer;
    }
}